bridge/engine/AudioRewriter.h

```diff
@@ -17,20 +17,20 @@ inline void rewrite(bridge::SsrcOutboundContext& outboundContext, const uint32_t
         outboundContext.rewrite.offset.timestamp = outboundContext.rewrite.lastSent.timestamp +
             codec::Opus::sampleRate / codec::Opus::packetsPerSecond - header.timestamp.get();
         outboundContext.rewrite.offset.sequenceNumber =
-            outboundContext.rewrite.lastSent.sequenceNumber + 1 - sequenceNumber;
+            static_cast<int32_t>(outboundContext.rewrite.lastSent.sequenceNumber + 1 - sequenceNumber);
         outboundContext.rewrite.sequenceNumberStart = sequenceNumber;
     }
 
-    header.sequenceNumber = header.sequenceNumber + outboundContext.rewrite.offset.sequenceNumber;
+    const uint32_t newSequenceNumber = sequenceNumber + outboundContext.rewrite.offset.sequenceNumber;
+    header.sequenceNumber = newSequenceNumber & 0xFFFFu;
     header.ssrc = outboundContext.ssrc;
     header.timestamp = outboundContext.rewrite.offset.timestamp + header.timestamp;
     header.payloadType = outboundContext.rtpMap.payloadType;
 
     outboundContext.rewrite.originalSsrc = originalSsrc;
-    if (static_cast<int32_t>(sequenceNumber + outboundContext.rewrite.offset.sequenceNumber -
-            outboundContext.rewrite.lastSent.sequenceNumber) > 0)
+    if (static_cast<int32_t>(newSequenceNumber - outboundContext.rewrite.lastSent.sequenceNumber) > 0)
```

Comment:
This seems does not take in account the rollover. if last is 0xFFFF and new is `0x0000` then the int32_t will be -1